### ISSUE TITLE
Bootstrap script doesn't preserve existing /etc/salt/minion

### ISSUE THREAD

**Issue by boboli** [saltstack/salt-bootstrap#830]
### Description of Issue/Question

When using the latest bootstrap script on a machine that already has salt installed, the existing /etc/salt/minion file gets overwritten by the default minion config file that comes with a fresh install.
### Setup

(Please provide relevant configs (Be sure to remove sensitive info).)
I tried this in a freshly provisioned vagrant ubuntu/trusty64 box.
### Steps to Reproduce Issue

(Include debug logs if possible, `bootstrap-salt.sh -D`.)
Start with a vagrant box with this Vagrantfile:

```
Vagrant.configure(2) do |config|
  config.vm.box = "ubuntu/trusty64"
  config.vm.provision :salt do |salt|
    salt.install_type = "git"
    salt.install_args = "v2015.8.7"
    salt.bootstrap_options = "-P"
  end
end
```

Then ssh into the vagrant box and edit the /etc/salt/minion file to something else. Maybe something like:

```
master: localhost
```

Then run:

```
$ curl -o bootstrap_salt.sh -L https://bootstrap.saltstack.com
$ sudo sh bootstrap_salt.sh git v2015.8.7
```

And see that the changes to /etc/salt/minion have been wiped.
### Versions and Systems

Bootstrap script was reporting:

```
 *  INFO: sh bootstrap_salt.sh -- Version 2016.04.18
```

**Comment by rallytime** [comment]
Thank you for letting us know, @boboli. I have fixed this issue with pull request #848.

**Comment by rallytime** [comment]
This fix was included in the most recent release of bootstrap, `2016.05.11`, which was cut yesterday.
